### PR TITLE
Initial framework for elbv2 implementation

### PR DESCRIPTION
**Pull request by tony-dot-sh**
This is the initial framework for `elbv2` usage. It acts more as a pass-through to `moto` right now, but it's expandable.

The one hitch I ran into was the service name `elbv2` vs the API endpoint being parsed and expecting `elasticloadbalancing` for it's URL response, so for that I added an `elif` in the edge service. Wasn't sure if I should've done something more with the starter or listener to register it properly as the endpoint URL vs the service name.

### CODE REVIEW

File: localstack/services/elbv2/elbv2_listener.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import re
+import logging
+from localstack.utils.common import to_str, to_bytes
+from localstack.services.generic_proxy import ProxyListener
+from moto.elbv2 import urls
+
+LOG = logging.getLogger(__name__)
+
+
+class ProxyListenerELBV2(ProxyListener):
```
Comment (whummer) [inline localstack/services/elbv2/elbv2_listener.py:10]: As we're currently not applying any logic in the proxy listener, I guess we can simply remove this file (and remove the `listener` argument from the `register_plugin(..)` call above).

File: tests/integration/test_elbv2.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        )
+
+        response = elbv2.create_load_balancer(
+            Name="".join(random.choice("0123abcd") for i in range(3)),
+            Subnets=[subnet0['Subnet']['SubnetId'], subnet1['Subnet']['SubnetId']],
+            Scheme="internal",
+            Tags=[{"Key": "key_name", "Value": "a_value"}],
+        )
+
+        lb = response.get("LoadBalancers")[0]
```
Comment (whummer) [inline tests/integration/test_elbv2.py:30]: Looks like we still have a few minor lint errors in this file (can be reproduced via `make lint` locally).
https://travis-ci.org/github/localstack/localstack/builds/759387845

**Review by whummer — COMMENTED** [review]
Looks great, thanks for adding this PR @tony-dot-sh ! 👍 Only added a few minor comments - can we please address them before merging? Thanks

**Comment by whummer** [comment]
Thanks for updating the PR @tony-dot-sh . Looks like we still have a minor issue in the tests - can you please take a look? https://travis-ci.org/github/localstack/localstack/builds/759793903
```
  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 312, in _parse_shape

    return handler(shape, node)

  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 174, in _get_text_content

    return func(self, shape, text)

  File "/opt/code/localstack/.venv/lib/python3.8/site-packages/botocore/parsers.py", line 486, in _handle_integer

    return int(text)

ValueError: invalid literal for int() with base 10: ''
```
Thanks!

**Comment by tony-dot-sh** [comment]
Ah, I may need to pull the latest since https://github.com/spulec/moto/pull/3694 was brought in recently.

**Comment by whummer** [comment]
Ah, I see - good catch. We'll pull the latest `moto` and will try to re-run the tests.. 👍 Thanks

**Comment by whummer** [comment]
Hi @tony-dot-sh , apologies for the delay on this one. Looks like the tests are still failing - is this something that needs to be fixed on moto side as well? Thanks for your help!

https://travis-ci.org/github/localstack/localstack/builds/759793903

**Comment by whummer** [comment]
@tony-dot-sh We've recently moved from Travis-CI to CircleCI. In order to trigger the CI builds, we'll need to close and re-create this PR - can you please re-create the PR, so we can take a look at the builds (and get any test errors fixed). Thanks!

**Pull request closed without merging**